package.json

```diff
@@ -23,42 +23,43 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@cruglobal/serverless-merge-config": "^1.0.1",
+    "@cruglobal/serverless-merge-config": "^1.1.0",
     "chance": "^1.0.16",
     "coveralls": "^3.0.2",
-    "factory-girl": "^5.0.2",
+    "factory-girl": "^5.0.3",
     "jest": "^23.0.0",
     "lodash": "^4.17.11",
     "sequelize-cli": "^4.1.1",
     "serverless": "^1.32.0",
-    "serverless-offline": "^3.25.12",
-    "serverless-plugin-bind-deployment-id": "^1.0.1",
+    "serverless-offline": "^3.31.2",
+    "serverless-plugin-bind-deployment-id": "^1.0.2",
     "serverless-plugin-scripts": "^1.0.2",
     "standard": "^10.0.3"
   },
   "dependencies": {
-    "aws-sdk": "^2.315.0",
+    "aws-sdk": "^2.352.0",
     "body-parser": "^1.18.3",
     "diff": "^3.5.0",
     "dotenv": "^5.0.0",
-    "express": "^4.16.2",
+    "express": "^4.16.4",
     "express-jwt": "^5.3.1",
     "glob": "^7.1.3",
+    "global-registry-nodejs-client": "github:CruGlobal/global-registry-nodejs-client",
     "hoek": "~> 4.2.1",
     "https": "^1.0.0",
     "jsonwebtoken": "^8.3.0",
-    "pg": "^7.4.3",
+    "pg": "^7.6.1",
     "pg-copy-streams": "^1.2.0",
     "pg-hstore": "^2.3.2",
     "promise-retry": "^1.1.1",
     "pug": "^2.0.3",
     "redis": "^2.8.0",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.5",
-    "rollbar": "^2.4.6",
-    "sequelize": "^4.38.1",
+    "rollbar": "^2.5.0",
+    "sequelize": "^4.41.1",
     "sequelize-paper-trail": "^2.0.0",
-    "serverless-http": "^1.6.0",
+    "serverless-http": "^1.8.0",
```

Comment:
lots of changes to dependency versions here that aren't really related to this commit. 
I'm not opposed to the changes, but could you put them in a separate commit please?

Reply:
looks like you decided to drop them from this PR. That's ok, as well.